src/3.1/Dockerfile

```diff
@@ -21,16 +21,21 @@ RUN apk add --no-cache --quiet \
     && mv /var/lib/neo4j/data /data \
     && chown -R neo4j:neo4j /data \
     && chmod -R 777 /data \
+    && mv /var/lib/neo4j/logs /logs \
+    && chown -R neo4j:neo4j /logs \
+    && chmod -R 777 /logs \
```

Comment:
why are we using `777` here but `755` later?

Reply:
use 755 everywhere now